cookbook/security/voters.rst

```diff
@@ -134,9 +134,9 @@ edit a particular object. Here's an example implementation::
                 return VoterInterface::ACCESS_DENIED;
             }
 
-            // double-check that the User object is the expected entity.
-            // It always will be, unless there is some misconfiguration of the
-            // security system.
+            // double-check that the User object is the expected entity
+            // it always will be unless there is some misconfiguration of the
```

Comment:
I agree. What about removing this part? `// it always will be unless there is some misconfiguration of the security system`